src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,44 @@
+.clock {
+  position: relative;
+  @include square (500px);
+
+  &__minute {
+    position: absolute;
+    left: 50%;
```

Comment:
You repeat those styles. I would suggest using either mixin or placeholder for this.